apps/account/templates/a4_candy_account/account_dashboard.html

```diff
@@ -40,7 +40,7 @@
                     <li class="dashboard-nav__page">
                         <a href="{% url 'socialaccount_connections' %}"
                            class="dashboard-nav__item dashboard-nav__item--interactive {% if request.resolver_match.url_name == "socialaccount_connections" %}is-active{% endif %}">
-                            <i class="far fa-users" aria-hidden="true"></i>
+                            <i class="fa fa-users" aria-hidden="true"></i>
```

Comment:
The new font awesome thing is to use far or fas. The old way still works, but we are menat to use the new thing. For users only the solid style (fas) is free.